connectors/sources/github.py

```diff
@@ -204,6 +204,9 @@ class GithubQuery(Enum):
             body
             state
             mergedAt
+            author {{
```

Comment:
@navarone-feekery, I think you might need to add the field in `_prepare_pull_request_doc` and `_prepare_issue_doc` too, otherwise it won't be reflected in the document indexed.

```
"author": pull_request.get("author", {}).get("login")
```
cc @seanstory

Reply:
I'm going to merge as this is approved, but if this is something you think warrants further discussion, let's talk in slack. We still have time to change it before release.